topic-operator/src/main/java/io/strimzi/operator/topic/TopicSerialization.java

```diff
@@ -132,10 +132,16 @@ private static int getPartitions(KafkaTopic kafkaTopic) {
     public static KafkaTopic toTopicResource(Topic topic, Labels labels) {
         ResourceName resourceName = topic.getOrAsKubeName();
         ObjectMeta om = topic.getMetadata();
+        Map<String, String> lbls = new HashMap<>();
+        lbls.putAll(labels.labels());
+        lbls.put(io.strimzi.operator.common.model.Labels.KUBERNETES_NAME_LABEL,
+            io.strimzi.operator.common.model.Labels.KUBERNETES_NAME);
+        lbls.put(io.strimzi.operator.common.model.Labels.KUBERNETES_INSTANCE_LABEL,
+            topic.getTopicName().toString());
+        lbls.put(io.strimzi.operator.common.model.Labels.KUBERNETES_MANAGED_BY_LABEL,
```

Comment:
why using the entire package name?

Reply:
Since there are two classes in scope with the same name `Labels` in both the `operator-common` and the `topic-operator` to call the correct one I need to use the full classpath name.
Is there another way to resolve the conflict of class names without using the full classpath?

Reply:
you are right, no unluckily there is no other mechanism as "aliases" for imports (as in C#).